src/init_game.cpp

```diff
@@ -282,7 +282,7 @@ int initGame()
 			}
 		}
 		FileIO::close(fp);
-		bookParser_t.createBooks(false);
```

Comment:
What was this doing? Why are we disabling it?

Reply:
(I would love to see a code review IDE that lets me explore the code more contextually...or I guess I can check out this branch next time I pick up this review)

Reply:
Good question, I had to disable this because it was trying to render and cache text from another thread which is not safe. Books are now "compiled" and cached offline and stored in a json file, so this line is not needed to run the game in release mode as long as you have that data file. You'd have to ask @WALLOFJUSTICE how this data file is created now (I think he just put it outside of this init thread).

Reply:
@WALLOFJUSTICE will a new, clean install of the game still automatically generate this cached file on first startup?

Reply:
Yes I had moved this below the asynchronous task, after loading completes. Takes 10 seconds or so. For some reason debug builds were OK on my machine,  but other machines/release mode would crash as the text cache changes data to render out the books. 

Thinking now, maybe could make temporary text cache just for this purpose that won't touch the Text() hashes used for the loading screen and it could solve the issue

https://github.com/WALLOFJUSTICE/Barony/commit/545b19f51e5c4113ee70b67140393dd8f0311d3e